### ISSUE TITLE
No item found

### ISSUE THREAD

**Issue by clbraver** [PoE-TradeMacro/POE-TradeMacro#165]
yesterday it worked great today I keep getting "No item found". everything is the same as before all I did was shut down my computer for the night.

**Comment by 852Foofer** [comment]
Getting the same. Using 1.4.0 beta. Using Firefox on fresh W10 install. Didn't change security settings. Deleted cookies in Firefox and Edge. Has stopped working since I restarted computer.

**Comment by itsmealwin** [comment]
Can confirm, same issue on my side. Restarted computer, stopped working..

**Comment by clbraver** [comment]
I fixed it by downloading the compiled version of the macro.
[https://github.com/PoE-TradeMacro/POE-TradeMacro/issues/164](url)

**Comment by itsmealwin** [comment]
@clbraver the url is 404

**Comment by clbraver** [comment]
should work now

**Comment by Eruyome** [comment]
That's kinda funny and weird... I have no idea why the compiled version would work in this case. Are you sure that the normal version doesn't work?

**Comment by clbraver** [comment]
I did a full reinstall of everything first autohotkeys and the plug-in. it still did not work. then I saw you post on the compiled version and tried that seeing nothing else worked.

**Comment by Eruyome** [comment]
What happen when you compile the script yourself? SImply right-click main.ahk -> Compile Script

**Comment by itsmealwin** [comment]
I tried the compiled version, did not work for me. Still same error pop up "Bypassing poe.trades CloudFlare protection failed!"

Compiling script myself and then running the main.exe does not work either. Same error pop up.

**Comment by Eruyome** [comment]
Please try the new release.

**Comment by itsmealwin** [comment]
@Eruyome I downloaded the new release 1.4.1 and following happens:

- Tried running the compiled exe and the error popped up again
- Deleted all cookies from internet explorer
- Deleted the cookies file in temp folder of the macro directory
- Run the "Run only this" ahk script
- Everything works again

**Comment by Eruyome** [comment]
Glad to hear, btw the temp dir is deleted on every script start (at least it should be, no idea if that can fail in any way).

**Comment by ZeusTTV** [comment]
Well i try it , thx

**Comment by 852Foofer** [comment]
After beta 1.4.0 returned 'no item found.' I went ahead and tried the 1.4.1 release, now I get the 'Bypassing poe.trades CloudFlare protection failed'-error message again. One thing I noticed is that the data in the cookie file is completely different from the data i extract manually (ctrl shift j). The compiled doesn't work either, btw. Again - security settings un-altered, cookies deleted manually multiple times. Used to work yesterday, has stopped working today.
